tests/BUILD

```diff
@@ -30,58 +31,6 @@ native_grakn_artifact(
     visibility = ["//tests:__subpackages__"],
 )
 
-py_test(
-    name = "test_concept",
-    srcs = [
-        "integration/base.py",
-        "integration/test_concept.py"
-    ],
-    deps = [
-        "//:client_python",
-    ],
-    data = [":native-grakn-artifact"],
-    args = ["$(location :native-grakn-artifact)"],
-    python_version = "PY3"
-)
-
-py_test(
-    name = "test_connection",
-    srcs = [
-        "integration/base.py",
-        "integration/test_connection.py"
-    ],
-    deps = [
-        "//:client_python",
-    ],
-    data = [":native-grakn-artifact"],
-    args = ["$(location :native-grakn-artifact)"],
-    python_version = "PY3"
-)
-
-py_test(
-    name = "test_query",
-    srcs = [
-        "integration/base.py",
-        "integration/test_query.py"
-    ],
-    deps = [
-        "//:client_python",
-    ],
-    size = "large",
-    data = [":native-grakn-artifact"],
-    args = ["$(location :native-grakn-artifact)"],
-    python_version = "PY3"
-)
-
-test_suite(
-    name = "test_integration",
-    tests = [
-        ":test_concept",
-        ":test_connection",
-        ":test_query",
-    ]
-)
```

Comment:
Now that we have fully functional BDD, we can throw away the old integration tests.